### PR TITLE
[CELEBORN-905] Redraw the flowchart backpressure.svg after worker pause logic is reconstructed

### PR DESCRIPTION
**Pull request by zwangsheng**
### What changes were proposed in this pull request?
Add a new `backpressure.svg` to replace the out-date one.

### Why are the changes needed?
After #1811, we refactor celeborn worker back-pressure logic, we should add new flowchart for user to understand.

### Does this PR introduce _any_ user-facing change?
Yes

### How was this patch tested?

backpressure

### CODE REVIEW

**Comment by waitinfuture** [comment]
LGTM. BTW, I think default 0.5 for `celeborn.worker.directMemoryRatioToResume` is too small, shall we change to 0.7?

**Review by cfmcgrady — APPROVED** [review]

**Comment by zwangsheng** [comment]
> LGTM. BTW, I think default 0.5 for `celeborn.worker.directMemoryRatioToResume` is too small, shall we change to 0.7?

+1 for setting default resume ratio to `0.7`, I tried set `0.65` on a production-scale cluster with good results. Sounds `0.7` is reasonable.

**Comment by waitinfuture** [comment]
> > LGTM. BTW, I think default 0.5 for `celeborn.worker.directMemoryRatioToResume` is too small, shall we change to 0.7?
> 
> +1 for setting default resume ratio to `0.7`, I tried set `0.65` on a production-scale cluster with good results. Sounds `0.7` is reasonable.

@zwangsheng Nice, so maybe we can change the default value in this PR and the svg can be changed at the same time.

**Comment by zwangsheng** [comment]
>  Nice, so maybe we can change the default value in this PR and the svg can be changed at the same time.

Updated.

File: docs/configuration/worker.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ license: |
 | celeborn.worker.directMemoryRatioForReadBuffer | 0.1 | Max ratio of direct memory for read buffer | 0.2.0 | 
 | celeborn.worker.directMemoryRatioToPauseReceive | 0.85 | If direct memory usage reaches this limit, the worker will stop to receive data from Celeborn shuffle clients. | 0.2.0 | 
 | celeborn.worker.directMemoryRatioToPauseReplicate | 0.95 | If direct memory usage reaches this limit, the worker will stop to receive replication data from other workers. This value should be higher than celeborn.worker.directMemoryRatioToPauseReceive. | 0.2.0 | 
-| celeborn.worker.directMemoryRatioToResume | 0.5 | If direct memory usage is less than this limit, worker will resume. | 0.2.0 | 
+| celeborn.worker.directMemoryRatioToResume | 0.7 | If direct memory usage is less than this limit, worker will resume. | 0.2.0 |
```
Comment (cfmcgrady) [inline docs/configuration/worker.md:42]: let's mention this change in the migration.md then

Comment (zwangsheng) [reply docs/configuration/worker.md:42]: Nice catch! I will add this change to `Upgrading from 0.3 to 0.4` part.

Comment (waitinfuture) [reply docs/configuration/worker.md:42]: @zwangsheng Let's make it happen in 0.3.1

**Comment by waitinfuture** [comment]
Thanks! Merged to main/0.3 using https://github.com/apache/incubator-celeborn/pull/1832

**Pull request closed without merging**